features/GEM111_No_duplicated_points_within_a_polyloop_or_polyline.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@GEM
+Feature: GEM111 - No duplicated points within a polyloop or polyline
+
+  Scenario: Agreement on no duplicated points within a polyloop
+
+      Given A file with Schema Identifier "IFC2X3"
```

Comment:
While this comes from the 2x3 impl agreements, I guess it's safe to assume it should apply files from all schema versions.

Reply:
Fair, removed this limitation.